app/src/main/java/com/amaze/filemanager/asynchronous/services/ftp/FtpService.kt

```diff
@@ -103,6 +105,14 @@ class FtpService : Service(), Runnable {
         return START_NOT_STICKY
     }
 
+    override fun onCreate() {
+        super.onCreate()
+
+        val powerManager = getSystemService(POWER_SERVICE) as PowerManager
```

Comment:
Please use `public final T getSystemService (Class<T> serviceClass)`, not the one that takes a string.

This [can be null](https://developer.android.com/reference/android/content/Context#getSystemService(java.lang.Class%3CT%3E)).